services/community-site/src/pages/Delegation/NodePage.tsx

```diff
@@ -0,0 +1,313 @@
+              </div>
+              <div className="percentageLegends">
+                <div className="percentageLegend">
+                  <div className="legendColor" style={{ background: '#15AC5B' }} />
+                  community-delegated
+                </div>
+                <div className="percentageLegend">
+                  <div className="legendColor" style={{ background: '#8E8E8E' }} />
+                  self-delegated
+                </div>
+                <div className="percentageLegend">
+                  <div className="legendColor" style={{ background: '#48BDFF' }} />
+                  available for delegation
+                </div>
+              </div>
+            </div>
+            <div className="delegationButtons">
+              <Button
+                onClick={() => setDelegateToNode(node)}
+                variant="contained"
+                color="secondary"
+                label="Delegate"
+              />
+              <Button
+                onClick={() => setUndelegateFromNode(node)}
+                variant="contained"
+                color="secondary"
+                label="Un-Delegate"
+              />
+            </div>
+          </div>
+        </div>
+        <hr className="nodeInfoDivider" />
+        <div className="delegatorsHeader">
+          <span className="delegatorsLegend">Delegators</span>
+          <div className="showOwnDelegation">
+            <Checkbox
+              value={delegationAtTop}
+              onChange={(e) => setDelegationAtTop(e.target.checked)}
+            />
+            Show my delegation at the top
+          </div>
+          <div className="delegatorsPagination">
+            <Button
+              className="paginationButton"
+              onClick={() => setDelegationPage(delegationPage - 1)}
+              disabled={delegationCount <= 20 || delegationPage === 1}
+              Icon={Icons.Left}
+            />
+            <Button
+              className="paginationButton"
+              onClick={() => setDelegationPage(delegationPage + 1)}
+              disabled={delegationCount <= 20 || delegationPage === delegationTotalPages}
+              Icon={Icons.Right}
+            />
+          </div>
+        </div>
+        <div className="tableHeader">
+          <span>Address</span>
+          <span>Amount of TARA delegated</span>
+        </div>
+        <div className="delegators">
+          {delegations.map((delegator, dIndex) => (
+            <div key={dIndex} className="delegatorRow">
+              <div className="address">
+                <span>{dIndex + 1 + offsetIndex}.</span> {delegator.address}
+              </div>
+              <div className="badges">
+                {delegator.isSelfDelegation && <div className="selfStake">self-stake</div>}
+                {delegator.isOwnDelegation && <div className="ownStake">your delegation</div>}
+              </div>
+              <div className="amount">{delegator.value}</div>
```

Comment:
Add `TARA` after the amount

Reply:
We should also use `ethers.utils.commify` on the value I think